wire-ios-mono.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -3,13 +3,31 @@
     "pins": [
       {
         "package": "DifferenceKit",
-        "repositoryURL": "https://github.com/wireapp/DifferenceKit",
+        "repositoryURL": "https://github.com/wireapp/DifferenceKit/",
         "state": {
           "branch": null,
           "revision": "eaf29c583df322d1d7c4c72c46a59eeb33bb54bf",
           "version": "1.3.0"
         }
       },
+      {
+        "package": "PINCache",
+        "repositoryURL": "https://github.com/pinterest/PINCache",
```

Comment:
Don't we usually create and reference forks of the original repos?

Reply:
I think so as well.

Reply:
Actually this is exactly the critics from our security engineer: no-one will update the libraries and patches are not used.
Why do you think it's a benefit to go with a fork?

Reply:
According to Francois the forking is done because:

1. We want to have a copy in case the (official) dependency is pulled by the authors
2. We want to have a place where the built binary can be stored to speed up the build process

I can sympathize with both reasons and I don't want to interfere with the way you do things.

All I ask is that if you _misuse_ forks this way to **make sure you regularly (days, not months or years) merge security relevant patches from upstream**. I don't about anything else really. Just the security relevant patches. 

If you fork to add features or patch stuff and upstream doesn't want to merge your patches, then of course it's sensible to use a fork. But even then: keep track of upstream changes relevant to security and merge them asap please.

We could open a bigger discussion around the fact that this strategy makes it harder to deliver a secure product because it breaks all existing ways to do security automation and is a very blunt and unorthodox way to achieve the stated objectives on top, but this is not the right place for it.

Reply:
I have synced our fork with `upstream` and use it now via Swift Package Manager. Please re-review, thanks.